app/controllers/users_controller.rb

```diff
@@ -26,7 +26,7 @@ def feed
     
     feed_likes = Like.all.where(fan_id: leader_ids).order(created_at: "asc")
```

Comment:
Unrelated to this PR, but `.all` is implicitly called before a `.where` so it can be safely removed.
```suggestion
    feed_likes = Like.where(fan_id: leader_ids).order(created_at: "asc")
```